src/oracle/monitor.js

```diff
@@ -0,0 +1,85 @@
+import {
+  boardSlottingMarshaller,
+  makeRpcUtils,
+  networkConfig,
+} from "../lib/rpc.js";
+import { readJSONFile, saveJSONDataToFile } from "../helpers/utils.js";
```

Comment:
not used

Reply:
Fixed in https://github.com/jacquesvcritien/agoric-cl-middleware/commit/0f2d4ec7dd638efe9fb0c21d9d11c8de0c3fb01e